less/about/history.less

```diff
@@ -1,19 +1,13 @@
-@import "./siteDetails.less";
```

Comment:
deprecated thanks to https://github.com/brave/browser-laptop/pull/10881/files#diff-237402632b025505a0517d846e9e32c1R34